packages/pangea-node-sdk/src/types.ts

```diff
@@ -5,13 +5,18 @@ import { Signer } from "utils/signer";
  */
 export interface ConfigOptions {
   domain?: string;
-  environment?: string;
+  environment?: ConfigEnv;
   requestRetries?: number;
   requestTimeout?: number;
   queuedRetryEnabled?: boolean;
   aqueuedRetries?: number;
 }
 
+export enum ConfigEnv {
```

Comment:
How is environment used? Is staging a needed option?

Reply:
I'm actually not entirely sure what this is used for. The enums used to be hard coded strings, so I thought it made sense to actually use a TS enum instead.